extras/ioc_hash_scan.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+#
+# This script will scan AWS instances for occurrences of IOC hashes listed in
+# the blob at src/ioc_scan/ioc_scanner.py
+#
+# The filename specified in the first argument
+# (instance-list-file) should contain a list of instance id strings, one per line.
+#
+# The following environmental varialbles must be set:
+# AWS_SHARED_CREDENTAILS_FILE
```

Comment:
Fix typos and prefer more standard "environment" terminology vs. "environmental":
```suggestion
# The following environment variables must be set:
# AWS_SHARED_CREDENTIALS_FILE
```